app/src/main/java/fr/free/nrw/commons/wikidata/model/DepictSearchResponse.java

```diff
@@ -0,0 +1,19 @@
+package fr.free.nrw.commons.wikidata.model;
+
+import java.util.List;
+
+/**
+ * Model class for API response obtained from search for depictions
+ */
+
+public class DepictSearchResponse {
+    private final List<DepictSearchItem> search;
+
+    public DepictSearchResponse(List<DepictSearchItem> search) {
+        this.search = search;
+    }
+
+    public List<DepictSearchItem> getSearch() {
```

Comment:
javadoc paraprasing what the method does, with other words

Reply:
added